composer.json

```diff
@@ -71,7 +71,7 @@
     "conflict": {
         "doctrine/dbal": "<2.13.1 || ^3.0 <3.2",
         "doctrine/mongodb-odm": "<2.3",
-        "doctrine/orm": "<2.10.2 || 2.16.0 || 2.16.1",
+        "doctrine/orm": "<2.14.0 || 2.16.0 || 2.16.1",
```

Comment:
```suggestion
        "doctrine/orm": "2.16.0 || 2.16.1",
```

Reply:
Here we need at least `2.14` to use the `EntityManager` constructor.

Reply:
My bad, I didn't see that the `"doctrine/orm": "^2.14.0"` constraint is in the `require-dev` section.
I'm sorry, I think you can use your original approach.

Reply:
So what's the plan here, keep the 2.14 in the require, but <2.10.2 in the conflict? Anything I can do?

Reply:
I should've kept the original `<2.14` conflict 😞, but let's solve it in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2639